Makefile

```diff
@@ -33,6 +33,19 @@ KIND_VERSION ?= 0.15.0
 KUBERNETES_VERSION ?= v1.25.0
 BATS_VERSION ?= 1.4.1
 
+# Binaries
+ENVSUBST_VER := v1.2.0
+ENVSUBST_BIN := envsubst
+ENVSUBST := $(TOOLS_DIR)/$(ENVSUBST_BIN)
+
+## --------------------------------------
+## Tooling Binaries
+## --------------------------------------
+
+$(ENVSUBST): $(TOOLS_MOD_DIR)/go.mod $(TOOLS_MOD_DIR)/go.sum $(TOOLS_MOD_DIR)/tools.go
+	cd $(TOOLS_MOD_DIR) && \
+	go build -o $(TOOLS_DIR)/$(ENVSUBST_BIN) github.com/a8m/envsubst/cmd/envsubst
```

Comment:
can we `tdnf install gettext` instead?

Reply:
if this is an issue for non-CI usecases, perhaps check the OS first?

Reply:
this approach makes the target reusable on local machine as well

Reply:
> if this is an issue for non-CI usecases, perhaps check if it's running in CI or check the OS first?

Is there a downside of using go to build the binary and using it from a `.tools` dir? With this, we can pin to a version and only install the binaries within the project instead of updating the existing binaries in CI or non-CI cases.